cmd/kube-dns/dns.go

```diff
@@ -25,11 +26,18 @@ import (
 	"k8s.io/kubernetes/pkg/version/verflag"
 )
 
+// Global kube-dns verbosity level.
+var logVerbosity = pflag.Bool("verbose-logs", false, "If set, logs are written at V(4), otherwise V(2).")
```

Comment:
usually you also need to set logtostderr, and there's some other module using --v for something else, so this was getting all related settings and turning verbosity all the way up. Turns out in this case logtostderr defaults to true, so just --v is enough.